packages/styles/scss/patterns/sub-patterns/card-link/index.scss

```diff
@@ -54,6 +54,10 @@
 
     &:hover {
       text-decoration: none;
+
+      .#{$prefix}--image__img {
+        filter: brightness(80%);
```

Comment:
Looks great, only needs the correct transition from the [@carbon/motion](https://www.npmjs.com/package/@carbon/motion) I believe.

Reply:
ah was wondering about why there was a small bit of lag on the brightness, will look into how they're using @carbon/motion

Reply:
ok I added the same transition I see in the carbon repo